package.json

```diff
@@ -1,25 +1,31 @@
 {
   "name": "penumbra-web",
-  "version": "1.0.0",
+  "private": true,
+  "license": "MIT OR Apache-2.0",
   "packageManager": "pnpm@8.6.11",
   "scripts": {
+    "all-check": "pnpm install && pnpm compile && pnpm format-check && pnpm lint && pnpm test && pnpm build && pnpm format-check:rust && pnpm lint:rust && pnpm test:rust",
     "build": "turbo run build",
+    "changeset": "changeset",
+    "changeset:publish": "changeset publish",
+    "clean": "turbo clean",
+    "clean:vitest-mjs": "find . -type f -name 'vite.config.ts.timestamp-*-*.mjs' -ls -delete",
     "compile": "turbo run compile",
     "dev": "turbo run dev  --concurrency 20",
+    "format": "syncpack format && prettier --write .",
+    "format-check": "syncpack lint && prettier --check .",
+    "format-check:rust": "turbo run format-check:rust",
     "lint": "turbo run lint -- --max-warnings=0",
-    "lint:rust": "turbo run lint:rust",
     "lint:fix": "turbo run lint -- --fix",
-    "test": "turbo run test",
-    "test:rust": "turbo run test:rust",
-    "clean": "turbo clean",
-    "clean:vitest-mjs": "find . -type f -name 'vite.config.ts.timestamp-*-*.mjs' -ls -delete",
-    "format": "prettier --write .",
-    "format-check": "prettier --check .",
-    "format-check:rust": "turbo run format-check:rust",
+    "lint:rust": "turbo run lint:rust",
     "playwright-install": "playwright install",
-    "changeset": "changeset",
-    "changeset:publish": "changeset publish",
-    "all-check": "pnpm install && pnpm compile && pnpm format-check && pnpm lint && pnpm test && pnpm build && pnpm format-check:rust && pnpm lint:rust && pnpm test:rust"
+    "test": "turbo run test",
+    "test:rust": "turbo run test:rust"
+  },
+  "pnpm": {
+    "overrides": {
+      "typescript": "^5.4.3"
+    }
```

Comment:
Why do we need this?

Reply:
oh, sorry, that isn't strictly necessary. it forces api-extractor to depend on the same version of typescript as our build, which had the benefit of making `vite-plugin-dts` shut up until a few days ago